pkg/virt-controller/services/net_binding_test.go

```diff
@@ -98,33 +102,39 @@ var _ = Describe("Network Binding", func() {
 			config := &v1.KubeVirtConfiguration{
 				NetworkConfiguration: &v1.NetworkConfiguration{},
 			}
-			_, err := services.NetBindingPluginSidecarList(vmi, config)
+			_, err := services.NetBindingPluginSidecarList(vmi, config, record.NewFakeRecorder(1))
 			Expect(err).To(HaveOccurred())
 		})
 	})
 
-	Context("binding plugin configuration", func() {
-		It("should read registered network binding plugins successfully", func() {
-			const testPluginName = "test"
-			testPlugin := v1.InterfaceBindingPlugin{SidecarImage: "kubevirt/network-slirp-binding"}
-
+	Context("network binding plugin configuration", func() {
+		It("should create Slirp hook sidecar with registered image (specified in Kubevirt config)", func() {
+			fakeRecorder := record.NewFakeRecorder(1)
+			vmi := v1.NewVMI("test", "1234")
 			config := &v1.KubeVirtConfiguration{
+				ImagePullPolicy: k8scorev1.PullIfNotPresent,
 				NetworkConfiguration: &v1.NetworkConfiguration{
 					Binding: map[string]v1.InterfaceBindingPlugin{
-						"plugin1":      {SidecarImage: "org1/plugin1"},
-						testPluginName: testPlugin,
-						"plugin2":      {SidecarImage: "org2/plugin2"},
+						"slirp": {SidecarImage: "kubevirt/network-slirp-plugin"},
 					},
 				},
 			}
 
-			Expect(services.ReadNetBindingPluginConfiguration(config, testPluginName)).To(Equal(&testPlugin))
+			Expect(services.SlirpNetBindingPluginSidecar(vmi, config, fakeRecorder)).To(Equal(hooks.HookSidecar{
+				ImagePullPolicy: k8scorev1.PullIfNotPresent,
+				Image:           "kubevirt/network-slirp-plugin",
+			}))
 		})
 
-		DescribeTable("should return no network binding plugins configuration, when config", func(config *v1.KubeVirtConfiguration) {
-			Expect(services.ReadNetBindingPluginConfiguration(config, "myplugin")).To(BeNil())
-		},
-			Entry("is nil", nil),
+		DescribeTable("should create Slirp hook sidecar with default image, when Kubevirt config",
+			func(config *v1.KubeVirtConfiguration) {
+				fakeRecorder := record.NewFakeRecorder(1)
+				vmi := v1.NewVMI("test", "1234")
+
+				Expect(services.SlirpNetBindingPluginSidecar(vmi, config, fakeRecorder)).To(Equal(hooks.HookSidecar{
+					Image: "quay.io/kubevirt/network-slirp-binding:20230830_638c60fc8",
```

Comment:
I think you should use `DefaultSlirpPluginImage` to express this is a specific fixed one.

Reply:
Done